components/Workshop/WorkshopListing.tsx

```diff
@@ -46,12 +50,18 @@ export default function WorkshopListing({
           user_booking={userBooking}
         />
       ) : (
-        <WorkshopListingSlotList
-          workshop={workshop}
-          slots={slots}
-          bookings={bookings}
-          user={user}
-        />
+        <Box>
+          <WorkshopListingSlotList
```

Comment:
I would maybe rename this to `WorkshopListingUpcomingSlotList` as we also have `WorkshopListingPastSlotList`

Reply:
I think tabs are a great idea! I can start working on that. What do you think about also including pagination there?

Reply:
> What do you think about also including pagination there?

That makes perfect sense :+1: